src/hooks/useImage.ts

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from 'react';
+import { FileRepository } from '@amityco/ts-sdk-react-native';
+
+interface UseImageProps {
+  fileId: string;
+  imageSize?: 'small' | 'medium' | 'large' | 'full';
```

Comment:
it would be better to use `enum` over string constraint

Reply:
Thanks. updated krub :)